testsuite/features/step_definitions/common_steps.rb

```diff
@@ -667,7 +667,7 @@
   # Prepare bootstrap script for different types of clients
   client = client_type == 'traditional' ? '--traditional' : ''
   node = get_target(host)
-  gpg_keys = get_gpg_keys(node)
+  gpg_keys = get_gpg_keys(node, target)
```

Comment:
Naming it is just a bit confusing now, as to obtain the node we use the function get_target, and then as second parameter of the next function we use the variable target extracted from few lines above 🤔
I just hope it is clear enough the next time we revisit this code

Reply:
I agree; there is room for improvement in the naming; it is worth improving it.